code/src/index.css

```diff
@@ -1,13 +1,449 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --dominantBlue: #707BFB;
+  --dominantWhite: #fff;
+  --accentBlue: #c6d9fa ;
+  --accentPink: #F3E0F0;
+  --accentBlack: #333;
+}
```

Comment:
A suggestion is to make this part easier to change by name it with more general names, like "primary/secondary and so on... Otherwise you have to change it throughout the code if you change color scheme to not include for example blue and pink.